### ISSUE TITLE
unstable.orbit datum gives no reading with an unstable orbit

### ISSUE THREAD

**Issue by jeu8** [bmad-sim/bmad-ecosystem#757]
I am running an optimization in Tao that uses the unstable.orbit datum. When I run the optimization, I lose the orbit but the unstable.orbit datum remains zero
[test.zip](https://github.com/bmad-sim/bmad-ecosystem/files/14083747/test.zip)

**Comment by DavidSagan** [comment]
Problem is not with the unstable.orbit datum but with the way Bmad handles the reference time calculation which uses a different reference orbit from the orbit that is tracked from the user given starting position (the ref time calc needs to be fixed and independent of the orbit the user wants to simulate). 

In this case, the optimizer is putting the lattice on the edge of tracking stability and what the unstable messages you see are actually from the plotting curve calc which is different from the datum value calc. So the unstable.orbit datum is zero since the tracking in the datum value calc is actually stable. 

The bottom line is that if you ignore the unstable messages you should be able to do the optimization. There is also a trick where you make the orbit you are simulating closer to the zero orbit of the reference time calc.
